manager/manager/settings.py

```diff
@@ -292,6 +292,8 @@
 if os.environ.get("HIDE_TRACE_TIMESTAMPS", False):
     TRACE_TIMESTAMPS = False
 
+# Check HeartBeat Commanded
+HEARTBEAT_QUERY_COMMANDED = os.environ.get("HEARTBEAT_QUERY_COMMANDED", 'false').lower() == 'true'
```

Comment:
This should be a variable to block heartbeat queries instead of allow them. This way the default behavior is to query the commander (so everything will keep working as expected) and when the variable is configured we avoid the query. Thus we can go then an set the env variable on the respective deployments to avoid excessive queries. Also the name should be something like "BLOCK_HEARTBEAT_QUERY".

Reply:
finally the name of variable is  HEARTBEAT_QUERY_COMMANDER in positive expression and the default value when it's not defined is true